data/remnant/remnant ships.txt

```diff
@@ -1107,18 +1094,6 @@ ship "Penguin"
 		"scramble protection" 0.25
 		"ion resistance" 0.05
 		"scramble resistance" 0.05
-		"thrust" 4.2
-		"turn" 80.0
-		"reverse thrust" 4.2
-		"flare sprite" "effect/remnant flare/small"
-			"frame rate" 5
-		"flare sound" "plasma tiny"
-		"steering flare sprite" "effect/remnant flare/small"
-			"frame rate" 5
-		"steering flare sound" "plasma tiny"
-		"reverse flare sprite" "effect/remnant flare/small"
-			"frame rate" 5
-		"reverse flare sound" "plasma tiny"
```

Comment:
For what reason are we nerfing the Penguin? Is it overpowered? Unbalanced somehow? Or do we just hate integrated engines for some reason?

Reply:
Those changes were removed from the PR before you commented this. I'm not actually sure _how_ you commented this on them.